src/pages/workspace/upgrade/UpgradeIntro.tsx

```diff
@@ -27,13 +27,14 @@ function UpgradeIntro({feature, onUpgrade, buttonDisabled, loading}: Props) {
     const {isExtraSmallScreenWidth, isSmallScreenWidth} = useResponsiveLayout();
     const {translate} = useLocalize();
     const iconSrc = feature.icon in Illustrations ? Illustrations[feature.icon as keyof typeof Illustrations] : Expensicon[feature.icon as keyof typeof Expensicon];
+    const iconType = feature === CONST.UPGRADE_FEATURE_INTRO_MAPPING.approvals ? CONST.ICON_TYPE_SQUARE : CONST.ICON_TYPE_AVATAR;
 
     return (
         <View style={styles.p5}>
             <View style={styles.workspaceUpgradeIntroBox({isExtraSmallScreenWidth, isSmallScreenWidth})}>
                 <View style={[styles.mb3, styles.flexRow, styles.justifyContentBetween]}>
                     <Avatar
```

Comment:
Maybe instead of using `Avatar` we should conditionally render `Icon` component only for approvals upgrade path?

Reply:
I'm not sure, wouldn't it be more invasive than adding special type?

Reply:
The svg we are trying to display isn't an avatar, so semantically it doesn't make sense to display it as one. That's why I would use a different component and diplsay it conditionally.

Ask Mateusz Titz what is more invasive, conditionally displaying a component or touching avatars logic 😆

Reply:
You can cleverly hide the logic in the `UpgradeIntro` props, this way it should be a clean adjustment!

Reply:
Just fixed it with conditionally adding borderRadius = 0